plugins/cloudservices/plugin.js

```diff
@@ -79,6 +84,10 @@
 					reqData.file = reqData.upload;
 					delete reqData.upload;
 
+					if ( !( fileLoader.customToken || editor.config.cloudServices_token ) ) {
+						CKEDITOR.error( 'cloudservices-token-error' );
+						return;
```

Comment:
I can see that request is still send even when error is reported:
![image](https://user-images.githubusercontent.com/1061942/33030943-fee538fa-ce1c-11e7-9dae-85a6c89f2c34.png)

maybe you should use `evt.cancel` here?